submissions/lalalilulu/memory-game/script.js

```diff
@@ -0,0 +1,102 @@
+//start the game
+(function launchApp() {
```

Comment:
why did you use IIFE here?

Reply:
to launch the app --> I have changed it, please see the updated version